easybuild/easyconfigs/p/Python/Python-2.7.10-intel-2015b.eb

```diff
@@ -113,7 +113,7 @@ exts_list = [
     ('pytz', '2015.4', {
         'source_urls': ['https://pypi.python.org/packages/source/p/pytz'],
     }),
-    ('pandas', '0.16.1', {
+    ('pandas', '0.16.2', {
```

Comment:
According to the policy we maintain of not bumping extension versions in existing easyconfigs, yes.